tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -65,8 +65,12 @@ if [[ -n "${IMAGE_CACHE_CONTAINER_REGISTRY_NAME}" ]]; then
 fi
 DOCKER_CMD="docker"
 
+
+if [[ $ORTMODULE_BUILD = true ]]; then
+    BUILD_EXTR_PAR=" --build_shared_lib ${BUILD_EXTR_PAR}"
```

Comment:
Just FYI: we don't use "--build_shared_lib" option in our python packaging pipelines. So please check if any one still want s to this use file for that purpose.  Probably not.

Reply:
Thanks for the notice. This line of change is out of date BTW.